configuration/jms-setup-wildfly.cli

```diff
@@ -0,0 +1,9 @@
+connect
+batch
+
+/subsystem=messaging/hornetq-server=default/address-setting=jms.queue.TriggerMetricCollectionQueue:add(redelivery-delay=1000, max-delivery-attempts=300)
```

Comment:
I just wonder if a hard limit 5 mins for delivery (300x 1000ms) is enough here. We could either consider much highter values (hours) or even set it to infinite (-5).

For newer WildFly/EAP versions (WF 10, EAP7), we can set up exponential backoff so that if delivery is really long, it does not try redelivery each second, but rather each 10s or so.